client/src/components/CreateContact.jsx

```diff
@@ -0,0 +1,110 @@
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import Snackbar from "@mui/material/Snackbar";
+import Alert from "@mui/material/Alert";
+import { useState } from "react";
+import { fetchContacts } from "../fetchContacts";
+
+const CreateContact = ({ setContacts }) => {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [phone, setPhone] = useState("");
+  const [notes, setNotes] = useState("");
```

Comment:
Here's a challenge for you - now that you're "comfortable" with 4 separate state variables for these 4 input fields, can we instead simplify the code and try using 1 state variable, an object that contains these 4 fields? Then the code would look a bit cleaner (and there'd be less code).